### ISSUE TITLE
capital letters for commands?

### ISSUE THREAD

**Issue by victorhck** [iggredible/Learn-Vim#72]
Hello!

translating the chapter 6, I found that you have changed the commands to capital letters. Let's pick up [this section](https://github.com/iggredible/Learn-Vim/blob/master/ch06_insert_mode.md#deleting-chunks-in-insert-mode) as example.

There you say:
```
Ctrl-H    Delete one character
Ctrl-W    Delete one word
Ctrl-U    Delete the entire line
```

Since that in Vim, for example, it's not the same i or I for insert text, shouldn't be more appropriate this?
```
Ctrl-h    Delete one character
Ctrl-w    Delete one word
Ctrl-u    Delete the entire line

just asking.

Greetings!
```

**Comment by iggredible** [comment]
Good question. I struggled with finding a consistent convention for the commands. 
One reason I switched the insert mode's Ctrl-h/w/u into uppercase is because the help itself uses uppercase (`:h i_CTRL-H`, `:h i_CTRL-W`, `:h i_CTRL-U`). I decided to leave them in uppercase to keep it consistent unless they are specifically required to be in lowercase (like `gu` vs `gU` or `i` vs `I` like you said).

In general, I try to follow whatever `:h` says.
